platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/platform/prometheus/AbstractMetricAdapter.java

```diff
@@ -16,19 +16,52 @@
 
 package com.swirlds.common.metrics.platform.prometheus;
 
+import static com.swirlds.common.metrics.platform.prometheus.NameConverter.fix;
+import static com.swirlds.logging.legacy.LogMarker.EXCEPTION;
+
+import com.swirlds.metrics.api.Metric;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.prometheus.client.SimpleCollector;
 import java.util.Objects;
 import java.util.concurrent.atomic.AtomicInteger;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public abstract class AbstractMetricAdapter implements MetricAdapter {
-
+    private static final Logger log = LogManager.getLogger(AbstractMetricAdapter.class);
```

Comment:
`Logger logger` is predominantly used in platform code.

Reply:
Thanks, @OlegMazurov  I should have searched what the pattern was. If it is ok with you, if there is any other suggestion that I should commit, I'll change the name in this PR, but if not, since platform-base is the owner of this file, I can create a follow-up task to replace the name that will require fewer approvals for merging it...

Reply:
Soved